### DISCUSSION TITLE
windows agent - application log issues

### DISCUSSION THREAD

**Discussion by adrianjdavies** [mondoohq/cnquery#3593] [General]
i have a windows agent:
reporting as Mondoo client version | 10.7.3
mondoo is reporting that the agent is updating at least in the last 24 hours

but its tripping up my fim/xdr

with these alerts: 

{"providerName":"cnspec","eventID":"1","version":"0","level":"2","task":"0","opcode":"0","keywords":"0x80000000000000","systemTime":"2024-03-18T06:46:02.7409139Z","eventRecordID":"17032","processID":"0","threadID":"0","channel":"Application","computer":"removed","severityValue":"ERROR"},"eventdata":{"data":"{\\\"level\\\":\\\"error\\\",\\\"error\\\":\\\"could not successfully complete scan: missing API endpoint\\\",\\\"time\\\":\\\"2024-03-18T06:46:02Z\\\"}"}}}

any ideas?

**Comment by chris-rock** [comment]
@adriandaviesmonek Thank you for the report. This error should only happen if no service account is defined. Has the client properly registered? You can validate that with `cnspec status`

**Comment by chris-rock** [comment]
I am closing this as there is missing feedback.
